File: Java/MathematicalPrograms/3DShapes/AreaAndPerimeterOfCuboid.java
import java.util.Scanner;
public class AreaAndPerimeterOfCuboid {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter a Length measure");
		int length = myObj.nextInt();
		System.out.println("Enter a Breadth measure");
		int breadth = myObj.nextInt();
		System.out.println("Enter a Height measure");
		int height = myObj.nextInt();
		int lengthbreadth = length*breadth;
		int lengthheight = length*height;
		int breadthheight = breadth*height;
		int tsa = 2*(lengthbreadth+lengthheight+breadthheight);
		int p = 4*(length+breadth+height);
		int lsa = 2*height*(length+breadth);
		System.out.println("Perimeter of the Cuboid is " + p + " cm");
		System.out.println("Total Surface Area of the Cuboid is " + tsa + " sq.cm.");
		System.out.println("Lateral Surface Area of the Cuboid is " + lsa + " sq.cm.");
	}
}

File: C++/MathematicalPrograms/3DShapes/AreaAndPerimeterOfCuboid.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	int Length, Breadth, Height;
	double perimeter, tsa, lsa;
	cout << "Enter a Length measure";
	cin >> Length;
	cout << "Enter a Breadth measure";
	cin >> Breadth;
	cout << "Enter a Height measure";
	cin >> Height;
	perimeter = 4*(Length+Breadth+Height);
	tsa = 2*((Length*Breadth)+(Length*Breadth)+(Breadth*Height));
	lsa = 2*Height*(Length+Breadth);
	cout << "Perimeter of the Cuboid is " << perimeter << " cm";
	cout << "\nTotal Surface Area of the Cuboid is " << tsa << " sq.cm.";
	cout << "\nLateral Surface Area of the Cuboid is " << lsa << " sq.cm.";
}